MVCDemos/Models/SampleData/DropDownButtonData.cs

```diff
@@ -29,5 +29,40 @@ public partial class DropDownButtonData {
             new SimpleData{ Text = "1.5", Value = "1.5" },
             new SimpleData{ Text = "2", Value = "2" }
         };
+        public static readonly IEnumerable<ListType> ListTypes = new[] {
+            new ListType{ Icon = "orderedlist", Alignment = "orderedlist", Hint = "Ordered" },
+            new ListType{ Icon = "bulletlist", Alignment = "bulletlist", Hint = "Bullet" }
+        };
+        public static readonly IEnumerable<SimpleData> ToolbarModes = new[] {
+            new SimpleData{ Text = "Multiline mode", Value = true },
+            new SimpleData{ Text = "Singleline mode", Value = false }
+        };
+        public static readonly IEnumerable<SimpleData> FontFamilies = new[] {
+            new SimpleData{ Text = "Arial", Value = "arial" },
+            new SimpleData{ Text = "Courier New", Value = "courier_new" },
+            new SimpleData{ Text = "Georgia", Value = "georgia" },
+            new SimpleData{ Text = "Impact", Value = "impact" },
+            new SimpleData{ Text = "Lucida Console", Value = "lucida_console" },
+            new SimpleData{ Text = "Tahoma", Value = "tahoma" },
+            new SimpleData{ Text = "Times New Roman", Value = "times_new_roman" }
+        };
+        public static readonly IEnumerable<FontStyle> FontStyles = new[] {
+            new FontStyle{ Icon = "bold", Hint = "Bold", Style = "bold" },
+            new FontStyle{ Icon = "italic", Hint = "Italic", Style = "italic" },
+            new FontStyle{ Icon = "underline", Hint = "Underlined", Style = "underline" },
+            new FontStyle{ Icon = "strike", Hint = "Strikethrough", Style = "strike" }
+        };
+        public static readonly IEnumerable<TextAlign> TextAligns = new[] {
+            new TextAlign{ Icon = "alignleft", Hint = "Align left", Alignment = "left" },
+            new TextAlign{ Icon = "aligncenter", Hint = "Center", Alignment = "center" },
+            new TextAlign{ Icon = "alignright", Hint = "Align right", Alignment = "right" },
+            new TextAlign{ Icon = "alignjustify", Hint = "Justify", Alignment = "justify" }
+        };
+        public static readonly IEnumerable<TextAlign> TextAlignsExtended = new[] {
+            new TextAlign{ Icon = "alignleft", Hint = "Align left", Alignment = "left", Text = "Align left" },
+            new TextAlign{ Icon = "aligncenter", Hint = "Center", Alignment = "center", Text = "Center" },
+            new TextAlign{ Icon = "alignright", Hint = "Align right", Alignment = "right", Text = "Align right" },
+            new TextAlign{ Icon = "alignjustify", Hint = "Justify", Alignment = "justify", Text = "Justify" }
+        };
```

Comment:
We cannot extend this model because it's used in another demo.

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/22b5a0be01fc7aac3b752346e4897831a0ccfafe
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/aecf62acb745d5b6f5750f7d85a550e527cd7692